code/src/components/Details.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react'
+import styled from "styled-components";
+import { useParams, useNavigate } from 'react-router-dom'
+import LoadingPage from './LoadingPage';
+
+const baseURL = "https://image.tmdb.org/t/p/";
+const imgSize = "original";
+
+const BackDrop = styled.div`
+        background-image: linear-gradient(180deg, rgba(0,0,0,0) 0%, rgba(0,0,0,1) 100%), url(${(props) => props.url});
+        background-size: cover;
+        min-height: 100vh;
+`;

```

Comment:
Would just like to say that the styling on this website is magnificent. I love the idea of a linear gradient, and having it visible all the time on mobile devices. This is just.... * chefs kiss *